cli-commands/shape/index.js

```diff
@@ -0,0 +1,42 @@
+const AsyncSoftExec = require('./../helpers/async-soft-exec');
+
+const Command = require('./../command');
+
+const commandMessages = require('./messages');
+const shapeCommandDefinition = require('./definition');
+
+const git = require('simple-git/promise')()
+
+// eoslime shape --name
+
+class ShapeCommand extends Command {
+    constructor() {
+        super(shapeCommandDefinition);
+    }
+
+    async execute(args) {
+        try {
+            commandMessages.StartShaping();
+            const optionsResults = await super.processOptions(args);
+            const shape = optionsResults.name;
+
+            if (!shape.repository) {
+                commandMessages.InvalidShapeName(shape.name);
+            }
+
+            await git.init();
+            await git.addRemote('origin', shape.repository);
+            await git.pull('origin', 'master');
+
+            const asyncSoftExec = new AsyncSoftExec('npm install');
```

Comment:
Don't need to install dependencies for the project -> just download it and the user will go through the documentation to read how to do it himself

Reply:
removed installation of project dependencies